sonic_platform_base/sonic_xcvr/api/public/cmisVDM.py

```diff
@@ -50,7 +50,9 @@ def get_vdm_page(self, page, VDM_flag_page):
         if page not in [0x20, 0x21, 0x22, 0x23]:
             raise ValueError('Page not in VDM Descriptor range!')
         vdm_descriptor = self.xcvr_eeprom.read_raw(page * PAGE_SIZE + PAGE_OFFSET, PAGE_SIZE)
-        
+        if not vdm_descriptor:
```

Comment:
There are other eeprom read in this function. can you take care of them also?

Reply:
fixed